task2/src/main/java/neutron/NeuronLearningTime.java

```diff
@@ -0,0 +1,24 @@
+package neutron;
+import java.util.function.Function;
+
+public class NeuronLearningTime extends Neutron {
+    public NeuronLearningTime(int numberOfInputs, Function<Double, Double> activationFunction) {
+        super(numberOfInputs, activationFunction);
+    }
+    public NeuronLearningTime(NeutronWorkingTime neutron) {
+        super(neutron.numberOfInputs, neutron.activationFunction);
+        weights = neutron.weights;
+    }
+
+    public double[] getWeights() {
```

Comment:
What's the point in writing there functions again for now if they do actually the same?